src/Autocomplete/CHANGELOG.md

```diff
@@ -11,6 +11,8 @@
 -   Added `tom-select/dist/css/tom-select.bootstrap4.css` to `autoimport` - this
     will cause this to appear in your `controllers.json` file by default, but disabled
     see.
+-   Allow passing `extra_options` key in an array passed as a `3rd` argument of the `->add()` method.
```

Comment:
Compared to [symfony/form changelog](https://github.com/symfony/symfony/blob/7.1/src/Symfony/Component/Form/CHANGELOG.md), i'd write it maybe more like 

- Add `extra_options` to XXX and YYY types (those options are .... to ...)